proto/identity/login.proto

```diff
@@ -25,6 +25,25 @@ service Identity {
       get: "/identity/debug"
     };
   }
+}
+
+service UserRegistryLogin {
+  rpc RequestLogin(UserLogin) returns (UserLogin) {
```

Comment:
Before you are tackling the login flow, I think it will be a lot easier if you start with routes that are similar to these
```
listAllUsers

getUser(pub_key)

getUser(name)

```

They will help you with a couple of things
1. Allow you to setup your service and routes. This would make this part of the PR a lot easier to review as well.
2. You will be forced to interact with the "eth" chain and be able to pull in useful information.

After that, you can tackle the logical part of the login mechanism. At that point, your server already has all the proper information it needs already to process a login.